addons/l10n_it_edi/models/account_move.py

```diff
@@ -232,6 +232,16 @@ def _l10n_it_edi_exempt_reason_tag_mapping(self):
     # Overrides
     # -------------------------------------------------------------------------
 
+    def copy(self, default=None):
+        default = dict(default or {})
+        new_moves = super().copy(default)
+
+        document_type = self.env['l10n_it.document.type'].search([]).grouped('code')
+        for old_move, new_move in zip(self, new_moves):
+            if new_move in old_move.debit_note_ids:
```

Comment:
This is the same, right? We don't need the original moves :thinking: 
```suggestion
        for new_move in new_moves:
            if new_move.debit_origin_id:
```

Reply:
Yes, it's the same. That way is better.